Octokit/Models/Response/PullRequest.cs

```diff
@@ -219,6 +220,11 @@ public bool Merged
         /// </summary>
         public bool Locked { get; protected set; }
 
+        /// <summary>
+        /// If users with write access to the base repository can push to the HEAD branch
```

Comment:
Could we reword this slightly?

"Whether maintainers of the base repository can push to the HEAD branch"